tests/streams/test_input.py

```diff
@@ -0,0 +1,70 @@
+from tests.output_betterproto.oneof import Test as TestOneof
+from tests.output_betterproto.nested import Test, TestNested, TestMsg, Sibling
+
+from src.betterproto.streams import CodedInputStream
+
+
+correct_oneof = TestOneof().from_dict(
+    {"pitied": 1, "just_a_regular_field": 123456789, "bar_name": "Testing"}
+)
+
+correct_nested = Test().from_dict(
+    {
+        "nested": {"count": 1},
+        "sibling": {"foo": 2},
+        "sibling2": {"foo": 3},
+        "msg": TestMsg.THIS,
+    }
+)
+
+
+def test_read_varint():
+    stream = open("tests/streams/varints.in", "rb")
```

Comment:
It is better to use a context manager instead of calling `.close()`:
```python
    with open("tests/streams/varints.in", "rb") as stream:
        ...
```

Reply:
Fixed